torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1994,11 +2008,13 @@ def create_symbol(
             self.var_to_sources[r].append(source)
         return r
 
-    # Generates a list of guards strings which, when evaluated in a context that
+    # Generates a tuple of 1. a list of guards strings which, when evaluated in a context that
```

Comment:
Maybe break 1. and 2. into separate lines, always find it hard to read otherwise.